frontend/src/containers/pages/ManageGroups/TopBar.jsx

```diff
@@ -0,0 +1,66 @@
+import Button from "@material-ui/core/Button";
+import { Link, useHistory } from "react-router-dom";
+
+import styles from "../../../styling/TopBar.module.scss";
+
+/**
+ * Component used for navigation and executing actions located at the top of the screen.
+ *
+ * @component
+ * @example
+ * const back = "/"
+ * const confirmModel = true
+ * return (
+ *   <TopBar back={back} confirmModel={confirmModel}>
+ *     { ... }
+ *   </TopBar>
+ * )
+ */
+export default function TopBar({
+  back = "/",
+  children = [],
+  confirmModal = false,
+}) {
+  const history = useHistory();
+
+  /**
+   * Function for changing variables to their appropriate states when the back button is pressed.
+   */
+  function handleLeaveAuthoringTool() {
+    history.push("/");
+  }
+
+  return (
+    <>
+      <div className={styles.topBar}>
+        <ul className={styles.leftTopBarList}>
+          <li className={styles.listItem}>
+            {confirmModal ? (
```

Comment:
nit: this might be never true? (dead code?)

Reply:
removed modal